starfish/test/test_apply.py

```diff
@@ -37,3 +37,19 @@ def test_apply_3d(synthetic_stack_factory):
     assert np.all(synthetic_stack.numpy_array == 1)
     synthetic_stack.apply(multiply, value=4, is_volume=True)
     assert (synthetic_stack.numpy_array == 4).all()
+
+
+def test_apply_gold_standard(gold_standard_dataset):
```

Comment:
why is this called the gold standard dataset?

Reply:
Because we have labels for it. I can probably think of a better wording... like "labeled_synthetic_dataset".

Reply:
I like that name better.